### DISCUSSION TITLE
asyncio support

### DISCUSSION THREAD

**Discussion by tallesl** [gitpython-developers/GitPython#1152] [Q&A] [answered]
I'm using the library to backup a bunch o repositories but, due to the quantity it takes a while to clone all of them.

Is it possible to clone them in parallel? Is there asyncio support?

**Comment by Byron** [comment, marked as answer]
Hi @tallesl ,

no, there is no asyncio support. The only way to parallelize GitPython would be to use the multiprocessing package.

However, if it's just about the initial clone or update, it might be easier to use `xargs`.

I am closing the issue, as I hope the above answers your question. Please feel free to keep posting here for further comments or follow-up questions.
